py7zr/compressor.py

```diff
@@ -1005,6 +1028,14 @@ class SupportedMethods:
             "filter_id": FILTER_BROTLI,
             "type": MethodsType.compressor,
         },
+        {
+            "id": COMPRESSION_METHOD.MISC_DEFLATE64,
+            "name": "DEFLATE64",
+            "native": False,
+            "need_prop": False,
+            "filter_id": FILTER_DEFLATE64,
```

Comment:
When compression is not supported,   ` "filter_id": FILTER_DEFLATE64,` should be `"filter_id": None`